db/src/fakes/purge_cache.js

```diff
@@ -101,7 +101,7 @@ class FakePurgeCache {
       });
   }
 
-  expire_cache_purges(exp) {
+  async expire_cache_purges(exp) {
```

Comment:
`async` is only needed for when the body of the function uses `await`. No need for this change.

Reply:
It doesn't hurt to keep them so I guess we can keep it.